src/test/fuzz/tx_pool.cpp

```diff
@@ -154,12 +154,15 @@ void CheckATMPInvariants(const MempoolAcceptResult& res, bool txid_in_mempool, b
         // It may be already in the mempool since in ATMP cases we don't set MEMPOOL_ENTRY or DIFFERENT_WITNESS
         Assert(!res.m_state.IsValid());
         Assert(res.m_state.IsInvalid());
+
+        const bool is_single_failure{res.m_state.GetResult() == TxValidationResult::TX_RECONSIDERABLE};
```

Comment:
After the name change, perhaps:

```suggestion
        const bool is_reconsiderable{res.m_state.GetResult() == TxValidationResult::TX_RECONSIDERABLE};
```

Reply:
PR description also needs to be updated to `TX_RECONSIDERABLE`

Reply:
taken